test/load/packer/README.md

```diff
@@ -0,0 +1,22 @@
+## Quick start
+
+# Consul AMI:
+
+Within the `consul-ami/` directory
+
+1) Update the `variables` section of the `consul.json` Packer template to configure the AWS region and datadog api key you would like to use. Feel free to reference this article to find your [datadog API key](https://docs.datadoghq.com/account_management/api-app-keys/#api-keys). 
```

Comment:
Is there some way to specify these as env vars or from the command line instead of editing a file?